plutus-core/executables/traceToStacks/Common.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE NamedFieldPuns   #-}
+{-# LANGUAGE TypeApplications #-}
+
+{-# OPTIONS_GHC -Wno-name-shadowing #-}
+
+module Common where
```

Comment:
We could put this in a separate component or something, but I think this is fine for now.